rskj-core/src/test/java/co/rsk/peg/performance/BtcBlockchainTest.java

```diff
@@ -1,103 +0,0 @@
-/*
```

Comment:
This class was an attempt by Angel to test his new methods, which did not use the framework, used dummy data and did not test what was really wanted. It can be maintained but generates more confusion than usefulness.

Reply:
thanks for the clarification, no need to keep it. Thanks